components/map.jsx

```diff
@@ -70,6 +71,40 @@ var MarkerPopup = React.createClass({
   }
 });
 
+var ClubStatusLabel = React.createClass({
+  propTypes: {
+    status: React.PropTypes.string.isRequired,
+    showApproved: React.PropTypes.bool
+  },
+  render: function() {
+    if (this.props.status === 'pending') {
+      return (
+        <span className="label label-warning"
+         title="This club is pending approval and is not visible to other users.">
+          pending
```

Comment:
Actually, they are the same right now, but there's a semantic difference between the two--`this.props.status` is the status code that the teach API gave us, and it'll always be `"pending"` even if we decide e.g. "pending approval" is a better phrase to show users. Also, the status code is independent of localization, so once we add l10n, we'll want to change the text we show users, but not `this.props.status`. So this looks like a DRY violation right now, but it's actually not, IMO :)